pandas_part.py

```diff
@@ -0,0 +1,53 @@
+from sqlalchemy import create_engine
+import pandas as pd
+import numpy as np
+import pyarrow as pa
+import pyarrow.parquet as pq
+
+from config import username, password, hostname, dbname, cols, path
+
+try:
+    engine = create_engine(path.format(username, password, hostname, dbname),
+                           echo=True)
+    conn = engine.connect()
+    df_users = pd.read_sql("select * from {}.users".format(dbname), conn)
+    df_companies = pd.read_sql("select * from {}.companies".format(dbname), conn)
+    df_development = pd.read_sql("select * from {}.development".format(dbname), conn)
+finally:
+    conn.close()
+    print('Table has been read. Connection is close')
+
+df_users['name'] = df_users[['name', 'surname']].agg(' '.join, axis=1)
+df_users = df_users.drop('surname', axis=1).rename(columns={'name': 'full_name'})
+pd.set_option('display.max_columns', None)
+df_users['birth_year'] = pd.to_datetime('1991-01-01 00:00:01')
+df_users['company_id'] = np.random.randint(1, 5, size=len(df_users))
```

Comment:
company_id should be randomly selected from the list of company ids and then assigned to users